maestro-test-scripts/src/main/groovy/multipoint/FixedRateTest.groovy

```diff
@@ -0,0 +1,143 @@
+/*
+ * Copyright 2018 Otavio R. Piske <angusyoung@gmail.com>
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package singlepoint
+
+@GrabConfig(systemClassLoader=true)
+
+@Grab(group='commons-cli', module='commons-cli', version='1.3.1')
+@Grab(group='org.apache.commons', module='commons-lang3', version='3.6')
+
+@Grab(group='org.msgpack', module='msgpack-core', version='0.8.3')
+
+@GrabResolver(name='Eclipse', root='https://repo.eclipse.org/content/repositories/paho-releases/')
+@Grab(group='org.eclipse.paho', module='org.eclipse.paho.client.mqttv3', version='1.1.1')
+
+@GrabResolver(name='orpiske-bintray', root='https://dl.bintray.com/orpiske/libs-release')
+@Grab(group='org.maestro', module='maestro-tests', version='1.3.0-SNAPSHOT')
+
+
+import org.maestro.client.Maestro
+import org.maestro.client.exchange.MaestroTopics
+import org.maestro.common.LogConfigurator
+import org.maestro.common.duration.TestDurationBuilder
+import org.maestro.reports.InspectorReportResolver
+import org.maestro.reports.InterconnectInspectorReportResolver
+import org.maestro.reports.ReportsDownloader
+import org.maestro.tests.MultiPointProfile
+import org.maestro.tests.rate.FixedRateTestExecutor
+import org.maestro.tests.rate.multipoint.FixedRateMultipointTestProfile
+
+maestroURL = System.getenv("MAESTRO_BROKER")
+if (maestroURL == null) {
+    println "Error: the maestro broker was not given"
+
+    System.exit(1)
+}
+
+senderBrokerURL = System.getenv("SENDER_BROKER_URL")
```

Comment:
Here, please use SEND_URL and RECEIVE_URL for the send/receive URLs in the multipoint